### PR TITLE
Bug fix in predict.py and README updates

### PR DESCRIPTION
**Pull request by ghost**
Hi Team,

This pull request contains changes based on local run in my environment. Changes are:
1. predict.py  - model.load_stage_dict was replaced with model.load_state_dict function
2. README file - minor corrections to give more context and accuracy for reproducibility

### CODE REVIEW

**Comment by neergaard** [comment]
Hi @shyam-lab, thanks for catching these bugs! I've merged them into `main.

And thank you for your interest in our work! Are you currently using it for another project?
